packages/uint/uint.2.0.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Andre Nathan <andre@digirati.com.br>"
+authors: [
+  "Andre Nathan <andre@digirati.com.br>"
+  "Jeff Shaw <shawjef3@msu.edu>"
+]
+license: "MIT"
+homepage: "https://github.com/andrenth/ocaml-uint"
+dev-repo: "git+https://github.com/andrenth/ocaml-uint.git"
+bug-reports: "https://github.com/andrenth/ocaml-uint/issues"
+synopsis: "Deprecated: An unsigned integer library"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name ] {with-doc}
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Your `dune-project` file says this package requires dune >= 1.11. Moreover, dune shouldn't be tagged as a build dependency anymore (see #14266)